src/views/Project/index.jsx

```diff
@@ -0,0 +1,168 @@
+    };
+  }
+  componentDidMount() {
+    const { tagRepoList } = this.state;
+
+    this.fetchGithubDataNext(tagRepoList);
+  }
+
+  fetchGithubDataNext(tagRepoList) {
+    const { client } = this.props;
+    let allQuery = '';
+
+    if (tagRepoList.length === 0) return;
+
+    tagRepoList.forEach((tag, idx) => {
+      const noCursorQuery = `_${idx}: search(first:100, type:ISSUE, query:"${
+        tag.query
+      }")\n
+      {
+      ...Issues
+      }\n`;
+
+      allQuery = allQuery.concat(noCursorQuery);
+    });
+
+    client
+      .query({ query: gql`{${allQuery}}\n${Issues}` })
+      .catch(
+        () =>
+          new Promise(resolve => {
+            resolve(false);
+          })
+      )
+      .then(({ data, error, loading }) => {
+        const repositoriesData = Object.entries(data).map(([key, value]) => ({
+          ...value,
+          ...tagRepoList[parseInt(key.split('_')[1], 10)],
+        }));
+        // issueData is formatted like data in the state
+        const issuesData = repositoriesData.reduce(
+          (previous, repoData) => [
+            ...previous,
+            ...repoData.nodes.map(issue => {
+              const obj = {
+                project: issue.repository.name,
+                id: issue.number,
+                description: `${issue.number} - ${issue.title}`,
+                tag: issue.labels.nodes.map(node => node.name).join(','),
+                lastupdate: issue.updatedAt,
+                assignedto: issue.assignees.nodes[0]
+                  ? issue.assignees.nodes[0].login
+                  : 'None',
+              };
+
+              return obj;
+            }),
+          ],
+          []
+        );
+
+        this.setState({
+          data: _.uniqBy(
+            [...this.state.data, ...issuesData],
+            'description'
+          ).sort((a, b) => (a.lastupdate > b.lastupdate ? -1 : 1)),
+          error,
+          loading,
+        });
+      });
+  }
+  render() {
+    const { projectInfo, loading, error, data } = this.state;
+
+    return (
+      <div>
+        <header>
+          <Grid
+            container
+            spacing={16}
+            direction="column"
+            justify="center"
+            alignItems="center">
+            <Grid item xs={12}>
+              <div>
+                <div />
+                <div>
+                  <Typography variant="display4" align="center">
+                    {projectInfo.name}
+                  </Typography>
+                  {projectInfo.description && (
+                    <Typography align="center">
+                      <ReactMarkdown source={projectInfo.description} />
+                    </Typography>
+                  )}
+                </div>
+                <div />
+              </div>
+            </Grid>
+          </Grid>
+          <BugsTable
+            projectName={projectInfo.name}
+            error={error}
+            loading={loading}
+            data={data}
+          />
```

Comment:
Let's not send the error and loading as props to `BugsTable`. We can handle them in this file similar to how it's done in https://github.com/taskcluster/taskcluster-web/blob/2998f6561a4de26908d4a2eba4aaca8443c15a53/src/views/Provisioners/ViewWorkerTypes/index.jsx#L114-L116.

For the error component, we can use https://github.com/taskcluster/taskcluster-web/blob/master/src/components/ErrorPanel/index.jsx.

For the loading icon, you can create a Spinner component with https://material-ui.com/demos/progress/#progress.